src/tlo/methods/cervical_cancer.py

```diff
@@ -0,0 +1,1802 @@
+        ),
+        "screening_max_age_hv_pos": Parameter(
+            Types.REAL, "maximum age individual to be screened if HIV positive"
+        ),
+        "yrs_between_screen_hv_pos": Parameter(
+            Types.REAL, "minimum years between screening if HIV positive"
+        ),
+        "yrs_between_screen_hv_neg": Parameter(
+            Types.REAL, "minimum years between screening if HIV negative"
+        ),
+        "palliative_care_bed_days": Parameter(
+            Types.REAL, "palliative_care_bed_days"
+        ),
+        "stage_1_3_daly_wt": Parameter(
+            Types.REAL, "stage_1_3_daly_wt"
+        ),
+        "stage_1_3_treated_daly_wt": Parameter(
+            Types.REAL, "stage_1_3_treated_daly_wt"
+        ),
+        "stage4_daly_wt": Parameter(
+            Types.REAL, "stage4_daly_wt"
+        ),
+        "min_yrs_between_screening_if_cin_treated": Parameter(
+            Types.REAL, "minimum years between screening if individual has been treated for CIN previously"
+        )
+    }
+
+    """
+    note: hpv vaccination is in epi.py
+    """
+
+    PROPERTIES = {
+        "ce_hpv_cc_status": Property(
+            Types.CATEGORICAL,
+            "Current hpv / cervical cancer status - note that hpv means persistent hpv",
+            categories=["none", "hpv", "cin1", "cin2", "cin3", "stage1", "stage2a", "stage2b", "stage3", "stage4"],
+        ),
+        "ce_date_diagnosis": Property(
+            Types.DATE,
+            "the date of diagnosis of cervical cancer stage (pd.NaT if never diagnosed)"
+        ),
+        "ce_stage_at_diagnosis": Property(
+            Types.CATEGORICAL,
+            "the cancer stage at which cancer diagnosis was made",
+            categories=[ "none", "stage1", "stage2a", "stage2b", "stage3", "stage4"],
+        ),
+        "ce_date_cin_removal": Property(
+            Types.DATE,
+            "the date of last cin removal (pd.NaT if never diagnosed)"
+        ),
+        "ce_date_treatment": Property(
+            Types.DATE,
+            "date of first receiving attempted curative treatment (pd.NaT if never started treatment)"
+        ),
+        "ce_ever_screened": Property(
+            Types.BOOL,
+            "whether ever been screened"
+        ),
+        "ce_ever_treated": Property(
+            Types.BOOL,
+            "ever been treated for cc"
+        ),
+        "ce_cured_date_cc": Property(
+            Types.DATE,
+            "ever cured of cervical cancer date"
+        ),
+        "ce_cc_ever": Property(
+            Types.BOOL,
+            "ever had cc"
+        ),
+            # currently this property has levels to match ce_hov_cc_status to enable the code as written, even
+            # though can only be treated when in stage 1-3
+        "ce_stage_at_which_treatment_given": Property(
+            Types.CATEGORICAL,
+            "the cancer stage at which treatment was given (because the treatment only has an effect during the stage"
```

Comment:
```suggestion

            "the cancer stage at which treatment was given (because the treatment  effect depends on the stage at which it is given)."

```